packages/yoga/src/Divider/Divider.theme.js

```diff
@@ -0,0 +1,3 @@
+const Divider = () => ({});
```

Comment:
I notice that there are some components that do not have a theme defined but we still let the file empty, we really need it, @coelhucas?

Reply:
For some reason my comments were lost 🤔 

---
This is something generated by our CLI, it indeed isn't needed in this and some other cases, but doesn't give us any real impact in the final bundling. Up to the developer keeping or not (although it feels "useless", a component missing this file could confuse a new collaborator who doesn't know how theming works here)

We're open to define a standard to this if you want :)